test/parser.test.ts

```diff
@@ -88,4 +88,16 @@ describe("parseRawArgs", () => {
       _: [],
     });
   });
+
+  it("handles arguments with no values", () => {
+    const args = ["--name", "--age"];
+    const opts = { string: ["name"], number: ["age"] };
```

Comment:
Can you add an (additional) test that ensures (by default) `--flag` sets to `true`

Reply:
Updated.